.github/workflows/build.yml

```diff
@@ -23,6 +23,7 @@ jobs:
           - 2.6.10
           - 2.5.8
         gemfile:
+          - rack_3
```

Comment:
We most probably would need to add a few exclusions of rack 3 in combination with some ruby versions down below in this file later.